apps/test/unit/promiseChainTest.js

```diff
@@ -1,197 +0,0 @@
-import sinon from 'sinon';
```

Comment:
I added this test a long time ago, hoping it would be useful documentation for testing with promises.  It turns out some of the behaviors it documented were Phantom-specific, and I haven't had to refer back to it - I think in general our testing practices with promises have gotten cleaner anyways, and now with async/await tests over promises have gotten easier to write too.  Since this isn't covering any production behavior, I propose we remove it.